.gitignore

```diff
@@ -24,6 +24,7 @@ vendor
 .DS_Store
 
 coverage.out
+coverage.txt
```

Comment:
This change is not required as file name for the coverage has been updated to covergae.out in [this PR](https://github.com/trustbloc/edge-service/pull/514/files#diff-bc37d034bad564583790a46f19d807abfe519c5671395fd494d8cce506c42947L26).